lightning/src/util/events.rs

```diff
@@ -343,6 +343,9 @@ pub enum Event {
 	/// This event is generated when a payment has been successfully forwarded through us and a
 	/// forwarding fee earned.
 	PaymentForwarded {
+		/// The channel between the source node and us. Optional because versions prior to 0.0.107
+		/// do not serialize this field.
+		channel_id: Option<[u8; 32]>,
```

Comment:
Lets call this `source_channel_id` or so? I imagine eventually we'll add a `sink_channel_id` field as well.